remappers/hostmetrics/cpu.go

```diff
@@ -132,6 +141,38 @@ func addCPUMetrics(metrics pmetric.MetricSlice, resource pcommon.Resource, datas
 			timestamp:   timestamp,
 			doubleValue: &softirqPercent,
 		},
+	)
+
+	// TODO (lahsivjar): Remove the dependency on the system.cpu.logical.count
+	// metric. The num of cores can be derived from the system.cpu.utilization
+	// metric using the cpu dimension.
+	if numCores == 0 {
+		return errors.New("system.cpu.logical.count metric is missing in the hostmetrics")
+	}
+
+	totalNorm := totalPercent / float64(numCores)
+	idleNorm := idlePercent / float64(numCores)
+	systemNorm := systemPercent / float64(numCores)
+	userNorm := userPercent / float64(numCores)
+	stealNorm := stealPercent / float64(numCores)
+	iowaitNorm := iowaitPercent / float64(numCores)
+	niceNorm := nicePercent / float64(numCores)
+	irqNorm := irqPercent / float64(numCores)
+	softirqNorm := softirqPercent / float64(numCores)
+
+	addMetrics(out, dataset, emptyMutator,
+		metric{
+			dataType:  pmetric.MetricTypeSum,
+			name:      "system.cpu.cores",
+			timestamp: timestamp,
+			intValue:  &numCores,
+		},
+		metric{
+			dataType:  pmetric.MetricTypeSum,
+			name:      "system.load.cores",
```

Comment:
This would be used for ?

Reply:
It is used in the load dasbhoard as a denominator in one of the panels.